ibm_catalog.json

```diff
@@ -1,54 +1,55 @@
 {
 	"products": [
 		{
-			"label": "Watson X - Generative AI",
-			"name": "Watson_X_-_Generative_AI",
+			"label": "Watsonx.ai SaaS with Assistant and Governance",
+			"name": "deploy-arch-ibm-watsonx-ai-saas",
 			"product_kind": "solution",
 			"tags": [
 				"watson",
-				"converged_infra"
+				"watsonx.ai",
+				"ibm_created"
 			],
 			"keywords": [
 				"watson",
-				"ai",
-				"compliance",
-				"fscloud",
-				"genai",
-				"ibmcloud",
-				"financial services",
 				"watsonx",
-				"llm",
-				"secure",
-				"secret manager",
-				"key protect",
-				"scc"
+				"ai"
 			],
-			"short_description": "Deploys a GenAI Pattern on IBM Cloud, including all underlying IBM Cloud infrastructure.",
-			"long_description": "This deployable architecture provides a comprehensive foundation for trust, observability, security, and regulatory compliance by configuring and deploying various services, including:\n- Configuring IBM Cloud Account: with recommended values meeting the [IBM Cloud Framework for Financial Services](https://cloud.ibm.com/docs/framework-financial-services?topic=framework-financial-services-about)\n- Deploying Observability Services: for application and platform logging and monitoring\n- Deploying Keys and Secrets Management Services: for storage and management of encryption keys and secrets\n- Deploying a Suite of Watson AI Services: to provide AI capabilities to the application \n\nThese configured and deployed services enable a secure and trustworthy deployment of Generative AI applications on IBM Cloud.\n\n# Objective\n\nThese pipelines facilitate the deployment of the application, vulnerability checks, and auditability, ensuring a secure and trustworthy deployment of Generative AI applications on IBM Cloud.",
-			"offering_docs_url": "https://github.com/terraform-ibm-modules/stack-ibm-genai/blob/main/README.md",
-			"offering_icon_url": "https://globalcatalog.cloud.ibm.com/api/v1/1082e7d2-5e2f-0a11-a3bc-f88a8e1931fc/artifacts/solution.svg",
+			"short_description": "Deploy Watsonx.ai SaaS with Assistant and Governance to automate the deployment and setup of the IBM watsonx platform.",
+			"long_description": "The Watsonx.ai SaaS with Assistant and Governance deployable architecture easily deploys and embeds AI across your business, manages all data sources, and accelerates responsible AI workflows on one platform.",
+			"provider_name": "FinOps Team.",
```

Comment:
should should be `IBM`

Reply:
Majority of catalog json is copy pated from the watson da to prevent overriding

Reply:
then it needs to be updated in theirs :) - can we reach out?